src/vyconf_pbt.ml

```diff
@@ -1137,21 +1149,25 @@ let rec encode_pb_request_rollback (v:request_rollback) encoder =
 let rec encode_pb_request_load (v:request_load) encoder = 
   Pbrt.Encoder.string v.location encoder;
   Pbrt.Encoder.key 1 Pbrt.Bytes encoder; 
+  Pbrt.Encoder.bool v.cached encoder;
+  Pbrt.Encoder.key 2 Pbrt.Varint encoder; 
```

Comment:
Since it's growing, maybe it's time to introduce a function like `opcode` that takes a sum type.